index.rst

```diff
@@ -11,9 +11,9 @@ BorgBackup – Deduplicating archiver with compression and authenticated encrypt
     BorgBackup (short: Borg) gives you:
 
     <ul>
-        <li>Space efficient storage of backups.
+        <li>Space, efficient storage of backups.
```

Comment:
The grammatical parenthesis go like this in that sentence: "(space efficient) (storage of backups)".

Reply:
As it is now, I don't know what point is trying to be conveyed.
`Space efficient storage of backups`
Should it be:
`Space efficient, storage of backups`?